lib/rules/require-padding-newlines-in-blocks.js

```diff
@@ -1,18 +1,22 @@
 /**
  * Requires blocks to begin and end with 2 newlines
  *
- * Types: `Boolean` or `Integer`
- *
- * Values: `true` validates all non-empty blocks,
- * `Integer` specifies a minimum number of statements in the block before validating.
+ * Types: `Boolean`, `Integer`, `Object`
+ *
+ * Values:
+ *  - `true` validates all non-empty blocks
+ *  - `Integer` specifies a minimum number of lines containing elements in the block before validating
+ *  - `Object` (at least one of properties must be true):
+ *      - `'open'`
+ *          - `true` validates that there is a newline after the opening brace in a block
+ *          - `false` ignores the newline validation after the opening brace in a block
+ *      - `'close'`
+ *          - `true` validates that there is a newline before the closing brace in a block
+ *          - `false` ignores the newline validation before the closing brace in a block
  *
  * #### Example
  *
  * ```js
- * "requirePaddingNewlinesInBlocks": true
```

Comment:
These were removed to keep the comments DRY. They were there because in the old documentation we do not explain the options parameter. With the new form of documentation, the values section actually provides a more descriptive version of this information. I think it looks better this way and is more consistent with all other rules that had only 1 example listed